skipper-mixin/README.md

```diff
@@ -0,0 +1,23 @@
+# Skipper Mixin
+
+The Skipper Mixin is a set of configurable, reusable, and extensible alerts and
+dashboards based on the metrics exported by the MySQLd Exporter. The mixin creates
```

Comment:
not clear to me: "MySQLd Exporter". We use https://github.com/prometheus/client_golang

Reply:
Ah sorry, this is a typo